airlock/views/request.py

```diff
@@ -629,13 +629,38 @@ def requests_for_workspace(request, workspace_name: str):
     requests_for_workspace = bll.get_requests_for_workspace(
         workspace_name, request.user
     )
+    request_filter = request.GET.get("status")
+    if request_filter:
+        filtered_requests = []
+        for r in requests_for_workspace:
+            if r.status.name == request_filter:
+                filtered_requests.append(r)
+
+        requests_for_workspace = filtered_requests
+
+    status_list = [
+        "PENDING",
+        "SUBMITTED",
+        "WITHDRAWN",
+        "APPROVED",
+        "REJECTED",
+        "PARTIALLY_REVIEWED",
+        "REVIEWED",
+        "RETURNED",
+        "RELEASED",
+    ]
+
+    filter_url = request.path
 
     return TemplateResponse(
         request,
         "requests_for_workspace.html",
         {
             "workspace": workspace_name,
             "requests_for_workspace": requests_for_workspace,
+            "request_filter": request_filter,
```

Comment:
If you change to use the status description, you'll also need to change this so it matches the filter labels; you can retrieve the enum from the name with `RequestStatus[request_filter]` and call `description()` on that